plugins/suggest/suggest.c

```diff
@@ -657,10 +657,22 @@ learner_init_columns(grn_ctx *ctx, grn_suggest_learner *learner)
   grn_obj *seqs, *events, *post_item, *items, *pairs;
 
   learner->seqs = seqs = grn_ctx_at(ctx, GRN_OBJ_GET_DOMAIN(learner->seq));
+  if (!learner->seqs) {
+    ERR(GRN_DOMAIN_ERROR,
```

Comment:
Why `GRN_DOMAIN_ERROR`?

It's based on `EDOM`:

>        EDOM            Mathematics argument out of domain of function
>                       (POSIX.1, C99).

http://man7.org/linux/man-pages/man3/errno.3.html